PM2024-001 - Two Kubernetes nodes went into NotReady state.md

```diff
@@ -0,0 +1,73 @@
+| 2024-03-13 10:36 | Terminated the unhealthy EC2 instance.                                                                                       |
+| 2024-03-13 12:10 | Incident declared resolved through statuspage.io, but investigations continued.                                              |
+| 2024-03-13 13:00 | Discovered that Grafana Agent was still down due to a newly applied LimitRange.                                              |
+| 2024-03-13 13:02 | Removed LimitRange from the namespace and restarted the Grafana Agent pod.                                                   |
+| 2024-03-13 13:49 | Concluded that a dev container using hangfire to trigger jobs, but fail with OOM. <br>Caused CrashLoopBackOff with new OOMs. |
+| 2024-03-13 13:55 | Discovered RDS issues for the instance used by the rogue dev container.                                                      |
+| 2024-03-13 13:55 | Found correlation between increased EC2 network traffic, RDS and the rogue dev container.                                    |
+| 2024-03-13 14:?? | Scaled the rough dev container down to 0 replicas. EC2 network traffic went down to normal levels after a few minutes.       |
+| 2024-03-13 14:?? | Scaled the rough dev container up to its original 3 replicas.                                                                |
+| 2024-03-13 15:03 | Alert triggered on Slack. The new node entered NotReady state.                                                               |
+| 2024-03-13 15:09 | Alert discovered.                                                                                                            |
+| 2024-03-13 15:15 | Scaled the rogue dev container to 0 replicas again.                                                                          |
+| 2024-03-13 15:18 | New node is added                                                                                                            |
+| 2024-03-13 15:19 | Email sent to the team owning the rogue dev container.                                                                       |
+| 2024-03-13 15:26 | Incident declared through statuspage.io.                                                                                     |
+| 2024-03-13 15:52 | Development team acknowledged the problem, and confirmed they were working on a fix.                                         |
+| 2024-03-14 09:20 | Incident closed.                                                                                                             |
+
+## Contributing Factors
+
+- Dev workloads mixed with prod workloads
+- 'grafana-agent' Not running on a dedicated node group unlike the majority of our monitoring stack
+
+## Action Items
+
+- Move 'grafana-agent' to monitoring node group
+- Add alerts for when bandwidth across the cluster reaches a certain threshold
+- Investigate if we can set limits memory limits on grafana-agent Helm chart
```

Comment:
set memory limits

Reply:
Nice catch, thanks 🙏

Reply:
Fixed in https://github.com/dfds/postmortems/pull/30/commits/6dc927b1b436108e1556843a8e05197c0770d542